R/Data-methods.R

```diff
@@ -319,6 +347,95 @@ setMethod(
   }
 )
 
+## DataOrdinal ----
+
+#' Updating `DataOrdinal` Objects
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' A method that updates existing [`DataOrdinal`] object with new data.
+#'
+#' @param object (`DataOrdinal`)\cr object you want to update.
+#' @param x (`number`)\cr the dose level (one level only!).
+#' @param y (`integer`)\cr the vector of toxicity grades (0, 1, 2, ...) for all
+#' patients in this cohort. You can also supply `numeric` vectors, but these
+#' will then be converted to `integer` internally.
+#' @param ID (`integer`)\cr the patient IDs.
+#'   You can also supply `numeric` vectors, but these will then be converted to
+#'   `integer` internally.
+#' @param new_cohort (`flag`)\cr if `TRUE` (default) the new data are assigned
+#'   to a new cohort.
+#' @param check (`flag`)\cr whether the validation of the updated object should
```

Comment:
this is a long argument description, hence I would suggest to move the details to a `@details` section and refer to that

Reply:
This was based on the parameter description in `update-Data`.  I have amended both as suggested.